bin/configs/spring-boot-3.yaml

```diff
@@ -0,0 +1,12 @@
+generatorName: spring
+outputDir: samples/openapi3/server/petstore/springboot-3
+inputSpec: modules/openapi-generator/src/test/resources/3_0/petstore.yaml
+templateDir: modules/openapi-generator/src/main/resources/JavaSpring
+additionalProperties:
+  groupId: org.openapitools.openapi3
+  documentationProvider: springdoc
+  artifactId: springboot
+  snapshotVersion: "true"
+  jakartaee: true
```

Comment:
add this to `cliOptions` to update generator doc

Reply:
I
- introduced a documented cli option, **_useSpringBoot3_**.
- renamed _jakartaee_ to **_useJakartaEe_** (still used internally, but not exposed as an option).

WDYT?

Reply:
@cachescrubber LGTM

Reply:
Ok.